tests/integration/components/live-join-test.js

```diff
@@ -169,4 +169,40 @@ module('Integration | Component | live-join', function (hooks) {
 
     assert.dom('[data-test-live-join-title]').containsText('Create a Event');
   });
+
+  test('it should prefill name in input box', async function (assert) {
+    const objToCheckFunctions = {
+      isInputHandler: assert.ok(true, 'inputHandler is working fine!'),
+      isClickHandlerWorks: assert.ok(true, 'clickHandler is working fine!'),
+      isBackHandlerWorks: assert.ok(true, 'backHandler is working fine!'),
+    };
+    this.setProperties({
+      role: 'host',
+      name: 'Jayasurya',
+      roomCode: '',
```

Comment:
This test explicitly sets the name and then checks if name is rendered. Can we instead set login data and check if first name is rendered?

Reply:
Controller name value assertion looks good to me.